forms-flow-components/src/components/CustomComponents/FormInput.tsx

```diff
@@ -46,7 +46,6 @@ export const FormInput: React.FC<FormInputProps> = ({
   const inputClassNames = `form-control-input ${icon ? 'with-icon' : ''} ${className}`;
 
   return (
-    <div className="form-input-box">
```

Comment:
This  was using as a parent class ,This may  override class written in other components with same name ?

Reply:
this parent class is removed from scss file so that child classes can be reused in formio case also @fahad-aot

Reply:
okay.